### PR TITLE
feature: added new API bind tcpsock:bind.

### PR DESCRIPTION
**Pull request by zhuizhuhaomeng**
Co-authored-by: doujiang24 <doujiang24@gmail.com>
Co-authored-by: lijunlong <zhuizhuhaomeng@gmail.com>

I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

**Comment by zhuizhuhaomeng** [comment]
This PR rebase #72  to the master and fixed test cases error.

File: README.markdown [hunk 1]
```diff
@@ -7693,6 +7695,42 @@ See also [ngx.socket.udp](#ngxsocketudp).
 
 [Back to TOC](#nginx-api-for-lua)
 
+tcpsock:bind
+------------
+**syntax:** *ok, err = tcpsock:bind(address)*
+
+**context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, ngx.timer.&#42;, ssl_certificate_by_lua&#42;*
```
Comment (doujiang24) [inline README.markdown:7702]: missing `ssl_session_fetch_by_lua*, ssl_client_hello_by_lua*`.

**Review by doujiang24 — APPROVED** [review]
otherwise, LGTM.

**Comment by zhuizhuhaomeng** [comment]
merged

**Pull request closed without merging**